metadata.json

```diff
@@ -15,47 +15,54 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 4.0.0 < 5.0.0"
+      "version_requirement": ">= 5.0.0 < 7.0.0"
     },
     {
       "name": "puppetlabs/cron_core",
       "version_requirement": ">= 1.0.3 < 2.0.0"
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 6.0.0 < 7.0.0"
+      "version_requirement": ">= 7.0.0 < 9.0.0"
     }
   ],
   "operatingsystem_support": [
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "8"
+        "7",
```

Comment:
This module was never intended for CentOS/RedHat 7, please remove this version.

Reply:
@fraenki done